Resources/Prototypes/_Goobstation/Loadouts/Jobs/Civilian/lawyer.yml

```
@@ -0,0 +1,95 @@
+  id: RSLawyer
+  storage:
+    back:
+      - RubberStampLawyer
+
+#Corporate lawyer stamps
+#20hr time is there to make sure lawyers know their jobs and have likely heard about the lore before.
+
+- type: loadout
+  id: RSCyberRep
+  effects:
+  - !type:JobRequirementLoadoutEffect
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)